steadybit/README.md

```diff
@@ -0,0 +1,63 @@
+# Steadybit
+
+## Overview
+
+[Steadybit](https://steadybit.com/) is the first reliability engineering platform to go beyond Chaos Engineering. Organizations in the eCommerce, SaaS and financial services industries, as well as global enterprises choose Steadybit to roll out chaos engineering across their teams to foster a culture of reliability. Steadybit provides everything an organization needs to make it safe and easy to bridge the gap between the experts (SREs, platform and expert teams) and the developers.
+
+The monitor status check step can be dragged&dropped into the experiment editor. Once done, you can use it to collect information about the state of the Datadog monitors and, optionally, to verify that they are within the expected status.
+
+Experiments can be aborted and marked as failed when the Datadog monitor status check's actual state diverges from the expected state. This helps implement pre-/post-conditions and invariants. For example, to only start an experiment when the system is healthy.
```

Comment:
We try and avoid marketing copy for integrations on the itnegrations page. Could you provide the following: 

1. What does Steadybit do in 1-2 sentences?
2. What will customers get from this integration? (the value of the integration) 
3. What data this integrations sends to datadog